stories/Containers/VendorsHeader.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Import } from '@styled-icons/boxicons-regular/Import';
+import { Add } from '@styled-icons/ionicons-sharp/Add';
+import { VendorsHeader, VendorsHeaderProps } from '../../src';
+import { createStoryTitle } from '../Constants';
+import { Story, Meta } from '@storybook/react';
+
+export default {
+    title: createStoryTitle('Vendors Header'),
+    component: VendorsHeader,
+} as Meta;
+
+export const defaultArgs = {
```

Comment:
You mentioned defaultArgs in the call - not sure if this is exactly the proper way?

Reply:
defaultArgs will collide with other defaultArgs make it unique and easy to use.
Ex: getVendorHeaderProps

Reply:
hopefully this change is better?